recipes/foundry-dev-tools/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "foundry-dev-tools" %}
+{% set version = "1.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/foundry-dev-tools-{{ version }}.tar.gz
+  sha256: 28decdbe2b6c269816cbe6912a5db942b44fa95129f729b667c39c10baa472cb
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8,<4.0
+    - setuptools >=46.1.0
+    - setuptools-scm >=5
+    - pip
+  run:
+    - python >=3.8,<4.0
+    - pyarrow
+    - pandas
+    - requests
+    - fs
+    - backoff
+    - palantir-oauth-client
+
+test:
+  imports:
+    - foundry_dev_tools
+    - transforms
```

Comment:
Should this be an installed Python package?

Reply:
The package installs foundry_dev_tools and the transforms module, yes.